packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-node.vue

```diff
@@ -235,10 +237,28 @@ Provide a consis summary in 100 words or less.
 			state.postForecastRunId = postSimId;
 			emit('update-state', state);
 
+			const datasetName = `Forecast run ${state.postForecastRunId}`;
+			const projectId = useProjects().activeProject.value?.id ?? '';
+			const datasetResult = await createDatasetFromSimulationResult(
+				projectId,
+				state.postForecastRunId,
+				datasetName,
+				false
+			);
+			if (!datasetResult) {
+				logger.error('Error creating dataset from simulation result.');
```

Comment:
Same comment as the other PR

Reply:
Removing log entirely as its already in the service file for us 
Good point